Patch-Source: https://github.com/rpy2/rpy2/commit/f50bf72d47a19dd6761ac212b54c0462f491e0bf
Adapted for the pypi source archive.
---
From f50bf72d47a19dd6761ac212b54c0462f491e0bf Mon Sep 17 00:00:00 2001
From: Laurent Gautier <lgautier@gmail.com>
Date: Tue, 22 Apr 2025 23:53:22 -0400
Subject: [PATCH] R release r 4 4 (#1167)

* Build and test with R-4.4 and R-release (4.5).

* initStdinHandler no longer visible with R-4.5.0.
---

diff --git a/rpy2/rinterface_lib/R_API_eventloop.h b/rpy2/rinterface_lib/R_API_eventloop.h
index 50edc98e5..f1e3f7ff0 100644
--- a/rpy2/rinterface_lib/R_API_eventloop.h
+++ b/rpy2/rinterface_lib/R_API_eventloop.h
@@ -28,8 +28,6 @@ typedef struct _InputHandler {
   
 } InputHandler;
 
-extern InputHandler *initStdinHandler(void);
-
 extern InputHandler *addInputHandler(InputHandler *handlers,
 				     int fd, InputHandlerProc handler,
 				     int activity);
